### ISSUE TITLE
AntiMicroX lacks Slope Data on Axis Input

### ISSUE THREAD

**Issue by rogerxx** [AntiMicroX/antimicrox#945]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

Antimicrox lacks reporting or adding slope data for axis input.

Many joystick to keyboard utilities appear to lack this feature, with sometimes the exception for mouse input. (eg. mouse acceleration)

### Describe the solution you'd like

Quite common for axis reporting sloped input rate, rather than direct one to one input rate.

In other words, an axis at minimal rate should report one to one input rate, an axis at maximum rate should report exponential input rate.  (eg 1 input = 1 output, 1 input = 10 output)

Most games expect a sloped input rather than a one to one input.  Without sloped data, a gas pedal is either on or off, while in the real world an axis reports a gradual increase, reporting a gradual minimum to gradual maximum output.
